docs/release.md

```diff
@@ -116,7 +116,7 @@ When considering making a bugfix release on the `v0.N.x` release cycle, the bug
     - Reverting a feature gate.
     - Changing the configuration to an easy to find value.
 2. The bug happens in common setups. To gauge this, maintainers can consider the following:
-    - The bug is not specific to an uncommon platform
+    - The bug is not specific to a platform on [Tier 2 or below](../docs/platform-support.md#tiered-platform-support-model).
```

Comment:
The intent here is to say that we will only do a bug fix for tier 1 platforms, right? Should we just say that instead of the double negative?

Reply:
The intent is to say that "if the bug is platform-specific, then we will only fix it for Tier 1 platforms". I guess I can word it just like that instead of the double negative